src/Bicep.Core/LanguageConstants.cs

```diff
@@ -17,12 +17,18 @@ public static class LanguageConstants
 
         public const string ListSeparator = ", ";
 
+        public const string TargetScopeKeyword = "targetScope";
         public const string ParameterKeyword = "param";
         public const string OutputKeyword = "output";
         public const string VariableKeyword = "var";
         public const string ResourceKeyword = "resource";
         public const string ModuleKeyword = "module";
 
+        public const string TargetScopeTypeTenant = "tenant";
```

Comment:
>public [](start = 8, length = 6)

It's fine for now, but we will need to figure out a way to make these things reside inside a namespace and not be part of the core language.

Reply:
Yeah, +1